examples/linear_model/plot_poisson_regression_non_normal_loss.py

```diff
@@ -194,9 +198,11 @@ def score_estimator(estimator, df_test):
         )
     )
 
+    return y_pred
+
 
 print("Constant mean frequency evaluation:")
-score_estimator(dummy, df_test)
+test_preds = [score_estimator(dummy, df_test)]
```

Comment:
If we go for the explicit Frequency column drop option, we will need to make sure to always drop it explicitly everywhere.